include/git2/notes.h

```diff
@@ -153,6 +153,33 @@ GIT_EXTERN(int) git_note_create(
 	const char *note,
 	int force);
 
+/**
+ * Add a note for an object from a commit
```

Comment:
Or maybe even better: adopt an optional notes reference which is to be written. This would ease finalizing the notes tree.